Can not connect to latest hf20 testnet

### Version of Python you are running ###
3.6.5


### Version of steem-python you are running ###
1.0.1



### Expected Behavior ###
Im expecting to connect to a node on the hf20 testnet (testnet-hf20-1535745600)


### Actual Behavior ###

The chain id seems to be not supported

### Steps to reproduce ###

> steempy info


### Stack Trace ###

Traceback (most recent call last):
  File "/usr/local/bin/steempy", line 11, in <module>
    load_entry_point('steem==1.0.1', 'console_scripts', 'steempy')()
  File "/usr/local/lib/python3.6/dist-packages/steem-1.0.1-py3.6.egg/steem/cli.py", line 776, in legacyentry
  File "/usr/local/lib/python3.6/dist-packages/steem-1.0.1-py3.6.egg/steem/steem.py", line 59, in __init__
  File "/usr/local/lib/python3.6/dist-packages/steem-1.0.1-py3.6.egg/steem/commit.py", line 96, in __init__
  File "/usr/local/lib/python3.6/dist-packages/steem-1.0.1-py3.6.egg/steem/wallet.py", line 61, in __init__
  File "/usr/local/lib/python3.6/dist-packages/steem-1.0.1-py3.6.egg/steem/steemd.py", line 69, in chain_params
AssertionError: The chain you are connecting to is not supported

--- a/steembase/chains.py
+++ b/steembase/chains.py
@@ -15,9 +15,9 @@ known_chains = {
         "sbd_symbol": "GBG",
         "vests_symbol": "GESTS",
     },
-    "TEST": {
+    "TESTS": {
         "chain_id":
-            "9afbce9f2416520733bacb370315d32b6b2c43d6097576df1c1222859d91eecc",
+            "46d82ab7d8db682eb1959aed0ada039a6d49afa1602491f93dde9cac3e8e6c32",
         "prefix":
             "TST",
         "steem_symbol":

